libbeat/common/reload/reload.go

```diff
@@ -95,34 +97,34 @@ func (r *registry) RegisterList(name string, list ReloadableList) error {
 }
 
 // MustRegister declares a reloadable object
-func (r *registry) MustRegister(name string, obj Reloadable) {
+func (r *Registry) MustRegister(name string, obj Reloadable) {
 	if err := r.Register(name, obj); err != nil {
 		panic(err)
 	}
 }
 
 // MustRegisterList declares a reloadable object list
-func (r *registry) MustRegisterList(name string, list ReloadableList) {
+func (r *Registry) MustRegisterList(name string, list ReloadableList) {
 	if err := r.RegisterList(name, list); err != nil {
 		panic(err)
 	}
 }
 
 // Get returns the reloadable object with the given name, nil if not found
-func (r *registry) Get(name string) Reloadable {
+func (r *Registry) Get(name string) Reloadable {
```

Comment:
Beware that a if a caller uses variable name `registry` this call will look like `registry.Get()` and it will look like it's going to return a new `Registry`. This is one of the cases where `GetReloadable` would be better.